src/tracker/face_id/identifier.py

```diff
@@ -147,28 +148,28 @@ def compute_known_embeddings(self):
                     "RGB"
                 )  # convert in RGB because png are RGBA
                 img_array = np.array(im)
-                uint8_tensor = (
-                    torch.from_numpy(img_array).permute(2, 0, 1).contiguous()
-                )
+                uint8_tensor = torch.from_numpy(img_array).permute(2, 0, 1).contiguous()
                 float32_tensor = uint8_tensor.to(dtype=torch.float32)
 
                 float32_tensor = float32_tensor.to(self.device)
                 face = self.detector.extract_face(float32_tensor)
                 if face is None:
-                    LOGGER.warning(f"Unable to find face in {directory}/{file}")
+                    LOGGER.warning(
+                        f"Image '{file}' for person '{directory}' must contain exactly one face. Skipping this image."
+                    )
                     continue
 
                 if self.debug:
                     save_tensor(face, f"{directory}.jpeg")
-                # TODO: check if there is only one face here
                 embed = self.feature_extractor.get_embedding(face)
                 embeddings.append(embed)
                 LOGGER.debug(f"Added embedding for {directory}/{file}")
 
             if not embeddings:
-                raise NoFacesDetectedError(
+                LOGGER.warn(
```

Comment:
nice, this also has the WARNING change, so once this PR is merged, you can close the other PR